src/test/java/aws/retrospective/controller/SectionControllerTest.java

```diff
@@ -0,0 +1,60 @@
+package aws.retrospective.controller;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.post;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import aws.retrospective.dto.CreateSectionRequest;
+import aws.retrospective.service.CommentService;
+import aws.retrospective.service.SectionService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+
+@WithMockUser("user1")
+@WebMvcTest(controllers = SectionController.class)
+class SectionControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private SectionService sectionService;
+    @MockBean
+    private CommentService commentService;
+
+    @Test
+    @DisplayName("신규 회고카드를 등록한다.")
+    void createSection() throws Exception {
+        //given
+        CreateSectionRequest request = CreateSectionRequest.builder()
+            .retrospectiveId(1L)
+            .templateSectionId(2L)
+            .sectionContent("내용")
+            .build();
+
+        //when //then
+        mockMvc.perform(
+                post("/sections")
+                    .content(objectMapper.writeValueAsString(request))
+                    .contentType(APPLICATION_JSON)
+                    .with(csrf())
+            )
+            .andDo(print())
+            .andExpect(status().isCreated())
+            .andExpect(jsonPath("$.code").value("201"))
+            .andExpect(jsonPath("$.message").doesNotExist());
```

Comment:
응답 값의 format을 일관되게 하기 위해서 `CommonApiResponse`를 사용하고 있다. 정상 응답 시에는 CommonApiResponse에 포함된 `message` 필드가 `null`이 저장된다.

테스트에서 `jsonPath("$.message").value(null))을 하니까, `Cannot invoke "org.hamcrest.Matcher.matches(Object)" because "matcher" is null java.lang.NullPointerException: Cannot invoke "org.hamcrest.Matcher.matches(Object)" because "matcher" is null` 예외가 발생한다.

왜냐하면 `$.message`를 통해 null에 접근을 시도하기 때문이다. 이를 해결 하기 위해서 `null`을 검증하기 위해서는 `value()`가 아닌 `doesNotExist()`를 사용해야 한다.